index.js

```diff
@@ -54,7 +81,23 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  //Return average of IMBD ratings which are strings that need to be converted into a number. Use Number() and add all ratings then divide by movies.length.
+  //If the inputted array is empty then return 0.
+
+  if(!movies || movies.length === 0) {
+    return 0;
+  };
+
+  let sumOfTotalImdbRatings = 0;
+  let averageTotalImdbRatings = 0;
```

Comment:
Absolutely love 😍 these names! Though we may not quite need `averageTotalImdbRatings` here